.eslintignore

```diff
@@ -0,0 +1,12 @@
+cdk/
```

Comment:
eslint was previously using the .gitignore file, but we now need to exclude the cdk directory from the main linter, so I've added a .eslintignore